### PR TITLE
[WIP] Upgrade v3 outbox sqlserver nhibernate EF sample

### PR DESCRIPTION
**Pull request by weralabaj**

### CODE REVIEW

File: samples/outbox/sqltransport-nhpersistence-ef/Version_3/Receiver/Web.config [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <configSections>
+        <!-- For more information on Entity Framework configuration, visit http://go.microsoft.com/fwlink/?LinkID=237468 -->
```
Comment (SimonCropp) [inline samples/outbox/sqltransport-nhpersistence-ef/Version_3/Receiver/Web.config]: Is this file required?

Comment (weralabaj) [inline samples/outbox/sqltransport-nhpersistence-ef/Version_3/Receiver/Web.config]: I'll need to double check, but I need to talk to Szymon to get it working.

File: samples/outbox/sqltransport-nhpersistence-ef/sample.md [hunk 2]
```diff
@@ -14,16 +14,24 @@ redirects:
...
+### Prerequisites
+ 1. Make sure you have SQL Server Express installed and accessible as `.\SQLEXPRESS`. 
+ 2. Create database called `nservicebus`.
+ 3. In the database create schemas `sender` and `receiver`. 
+ 4. The [Outbox](/nservicebus/outbox) feature is designed to provide *exactly once* delivery guarantees without the Distributed Transaction Coordinator (DTC) running. Disable the DTC service to avoid seeing warning messages in the console window. If the DTC service is not disabled, you will see `DtcRunningWarning` message when the sample project is started. 
+
+
+### Running the project
+ 1. Start the Sender project (right-click on the project, select the `Debug > Start new instance` option).
+ 2. In the Sender's console window you should see `Press <enter> to send a message` text.
```
Comment (SimonCropp) [inline samples/outbox/sqltransport-nhpersistence-ef/sample.md]: remove you

File: samples/outbox/sqltransport-nhpersistence-ef/sample.md [hunk 3]
```diff
@@ -14,16 +14,24 @@ redirects:
...
- 6. Hit <enter>.
- 7. On the Receiver console you should see that order was submitted.
- 8. On the Sender console you should see that the order was accepted.
- 9. Finally, after a couple of seconds, on the Receiver console you should see that the timeout message has been received.
- 10. Open SQL Server Management Studio and go to the receiver database. Verify that there is a row in saga state table (`dbo.OrderLifecycleSagaData`) and in the orders table (`dbo.Orders`)
+### Prerequisites
+ 1. Make sure you have SQL Server Express installed and accessible as `.\SQLEXPRESS`. 
+ 2. Create database called `nservicebus`.
+ 3. In the database create schemas `sender` and `receiver`. 
+ 4. The [Outbox](/nservicebus/outbox) feature is designed to provide *exactly once* delivery guarantees without the Distributed Transaction Coordinator (DTC) running. Disable the DTC service to avoid seeing warning messages in the console window. If the DTC service is not disabled, you will see `DtcRunningWarning` message when the sample project is started.
```
Comment (SimonCropp) [inline samples/outbox/sqltransport-nhpersistence-ef/sample.md]: remove you

**Comment by SimonCropp** [comment]
@weralabaj apart from some minor comments this looks good to go. is there anything u need from me to move it out of the  WIP state?

**Comment by weralabaj** [comment]
I get timeout exceptions, but after talking to Szymon and Szymon++ I opened a separate issue to handle it https://github.com/Particular/NServiceBus/issues/3469. IMO good to merge.

File: samples/outbox/sqltransport-nhpersistence-ef/Version_3/Receiver/Web.config (hunk 1 above)
Comment (SimonCropp) [inline samples/outbox/sqltransport-nhpersistence-ef/Version_3/Receiver/Web.config]: @weralabaj did u check this with @SzymonPobiega ?

**Comment by weralabaj** [comment]
Because of timeouts issue waiting for merging potential fix https://github.com/Particular/NServiceBus.NHibernate/pull/154.

File: samples/outbox/sqltransport-nhpersistence-ef/Version_3/Receiver/Web.config (hunk 1 above)
Comment (weralabaj) [inline samples/outbox/sqltransport-nhpersistence-ef/Version_3/Receiver/Web.config]: No need, I removed them and seems to work like before. I think it was some left-over from the previous version.

**Comment by SimonCropp** [comment]
@weralabaj i am going to merge this one. i know there are timeout issues. but i think they have been adequately captured elsewhere and will be pushed back down when we do a nuget update. so no need to keep it open

**Comment by weralabaj** [comment]
Yes, the reason why I didn't want to merge it yet is that if anybody downloads this sample they"ll get exceptions :/
But I hope we'll deal with those issues soon, so I'll make sure to update references in samples.

**Comment by SimonCropp** [comment]
@weralabaj well the v6 samples are all clearly marked with unstable so i am not overly concerned
